oe-init/internal/gen-oe-setup.py

```diff
@@ -97,6 +98,14 @@ def main():
 	ss_dir = os.path.abspath(ss_dir)
 	print_info("Selected shared state dir: " + ss_dir)
 
+	if args.pers_dir == "default":
+		pers_dir = os.path.join(root_path, "persistance")
```

Comment:
I think this should be kept as `cache` as is the default directory name when stored under the build directory

Reply:
Updated in new commit